Makefile

```diff
@@ -1,18 +0,0 @@
-SRC = $(wildcard src/*.js)
```

Comment:
Issues with the Makefile that could be handled better from package.json:

* **clean** is removing the src directory, feels like you moved from a transpiler to native compile and never fixed that.
* **test** is just running mocha and is relying on a path to get to it see npm test in package.json now
* **coverage** is just running istanbul which again is relying on a special node_modules path to be very specific see npm run coverage now.

Am I missing another reason you have this file?